tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/ReportAPI.java

```diff
@@ -108,6 +114,87 @@ public Response getReport(
                 .getReportStatus(user, CLDRLocale.getInstance(locale))).build();
     }
 
+
+    @GET
+    @Path("/locales/{locale}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary="List vetting results of one or more locales")
+    @APIResponse(
+        responseCode = "200",
+        description = "list responses",
+        content = @Content(mediaType = "application/json",
+            schema = @Schema(implementation = LocaleReportVettingResults.class))
+    )
+    public Response getReportLocaleStatus(
+        @HeaderParam(Auth.SESSION_HEADER) String session,
+        @Parameter(required = true, example = "en",
+            schema = @Schema(type = SchemaType.STRING, description = "Locale ID or '-' for all")) @PathParam("locale") String locale) throws SQLException {
+        final CookieSession mySession = Auth.getSession(session);
+        if (mySession == null) {
+            return Auth.noSessionResponse();
+        }
+        Iterable<CLDRLocale> locales = null;
+        CLDRLocale onlyLoc;
+        if (locale.equals("-")) {
+            locales = SurveyMain.getLocalesSet();
+            onlyLoc = null;
+        } else {
+            onlyLoc = CLDRLocale.getInstance(locale);
+            locales = Collections.singleton(onlyLoc);
+        }
+        LocaleReportVettingResults r = new LocaleReportVettingResults();
+        // make a copy of the DB subset here, for performance.
+        VoterReportStatus<Integer> db = ReportsDB.getInstance().clone(null, onlyLoc);
+        VoteResolver<ReportAcceptability> res = new VoteResolver<>();
+        // set of all valid userids
+        final Set<Integer> allUsers = CookieSession.sm.reg.getVoterToInfo().keySet();
+        for (final CLDRLocale loc : locales) {
+            LocaleReportVettingResult rr = new LocaleReportVettingResult();
+            rr.locale = loc.toString();
+            for (final ReportId report : ReportId.values()) {
+                db.updateResolver(loc, report, allUsers, res);
+                rr.reports.add(new ReportVettingResult(report, res));
+            }
+            r.locales.add(rr);
+        }
+        return Response.ok(r).build();
+    }
+
+
+    public static class LocaleReportVettingResults {
+        public LocaleReportVettingResults() {}
+        private Set<LocaleReportVettingResult> locales = new HashSet<LocaleReportVettingResult>();
+        public LocaleReportVettingResult[] getLocales() {
+            return locales.toArray(new LocaleReportVettingResult[0]);
+        }
+    }
+
+    public static class LocaleReportVettingResult {
+        public String locale;
+        private Set<ReportVettingResult> reports = new HashSet<ReportVettingResult>();
+        public ReportVettingResult[] getReports() {
+            return reports.toArray(new ReportVettingResult[0]);
+        }
+    }
+
+    public static class ReportVettingResult {
+        // @Schema(description = "raw VoteResolver output")
+        // public String raw;
```

Comment:
commented out; not needed?

Reply:
for other voting items we do pass VoteResolver.toString() to the client for debugging how the voting turned out, this could be enabled here also.